### PR TITLE
Modifying 'taco platform list' to include remote platforms

### PR DESCRIPTION
**Pull request by MSLaguana**

### CODE REVIEW

File: src/taco-cli/cli/utils/platformPluginCommandBase.ts [hunk 1]
```diff
@@ -146,6 +153,9 @@ export class PlatformPluginCommandBase extends commands.TacoCommandBase {
             return cordovaWrapper.invokePlatformPluginCommand(self.name, self.cordovaCommandParams, data);
         })
             .then(function (): Q.Promise<any> {
+            return self.postInvokeStep(data);
```
Comment (nisheetjain) [inline src/taco-cli/cli/utils/platformPluginCommandBase.ts]: not sure if I follow, why do we need to execute add/remove related code like specsToPersist for "list" command? can we use existing pattern like "run" and invoke list via CordovaWrapper?

File: src/taco-cli/cli/platform.ts [hunk 2]
```diff
@@ -127,13 +129,24 @@ class Platform extends commandBase.PlatformPluginCommandBase {
...
 
             case CommandOperationStatus.Success: {
-                logger.logLine();
                 this.printSuccessMessage(platforms, operation);
                 break;
             }
         }
     }
 
+    public postInvokeStep(data: commands.ICommandData): Q.Promise<any> {
```
Comment (meenakb) [inline src/taco-cli/cli/platform.ts]: How about a better name for this method?

**Comment by meenakb** [comment]
:+1:  otherwise

File: src/taco-cli/cli/utils/platformPluginCommandBase.ts (hunk 1 above)
Comment (MSLaguana) [inline src/taco-cli/cli/utils/platformPluginCommandBase.ts]: Currently all "taco platform X" and "taco plugin X" commands go through this class doing the same steps, eventually deferring to corodvaWraper.invokePlatformPluginCommand just above. If we want to refactor these classes to use the subcommand style that the other commands use, it'll be a bigger task.

Comment (nisheetjain) [inline src/taco-cli/cli/utils/platformPluginCommandBase.ts]: can we do something like this in run module? 
    public run(data: commands.ICommandData): Q.Promise<ICommandTelemetryProperties> {
        if (data.options.remain == "--list") {
            // run list command logic 
        } else {
        return super.run(data);
        }
    }

File: src/taco-cli/cli/platform.ts (hunk 2 above)
Comment (joshuaskelton) [inline src/taco-cli/cli/platform.ts]: :+1:

**Pull request closed without merging**